testimony/__init__.py

```diff
@@ -428,17 +428,28 @@ def validate_docstring_report(testcases):
                 if not SETTINGS['tokens'][token].validate(value):
                     invalid_token_values.setdefault(token, value)
             if invalid_token_values:
+                invalid_token_strings = []
+                for key, value in sorted(invalid_token_values.items()):
+                    settings_token = SETTINGS['tokens'][key]
+                    # TODO: Rework constants for token type to not compare bare string
+                    if settings_token.token_type == 'choice':
+                        valid_choices_string = ", choices: {}".format(settings_token.choices)
+                    else:
+                        valid_choices_string = ''
+                    # with #149 in I can also include case sensitivity boolean in this output
```

Comment:
I am waiting for travis so we can get #149 merged and then we could consider implementing the case sensitivity output here. That would be awesome.

Reply:
Just merged https://github.com/SatelliteQE/testimony/pull/149 as Travis is taking forever to complete

Reply:
Rebased and included case sensitivity reference. I need to update sample output and will keep this in draft until travis completes cleanly